tf2_ros/src/buffer.cpp

```diff
@@ -44,10 +44,22 @@
 #define ROS_INFO printf
 #define ROS_WARN printf
 
-
 namespace tf2_ros
 {
 
+// Added to backport: https://github.com/ros2/geometry2/pull/281
+std::mutex g_object_map_to_cb_handle_mutex;
+std::map<void*, std::unordered_map<TimerHandle, tf2::TransformableCallbackHandle>> g_object_map_to_cb_handle;
+
+void deleteTransfromCallbackHandle(void *class_ptr, const TimerHandle &timer_handle)
```

Comment:
```suggestion
void deleteTransformCallbackHandle(void *class_ptr, const TimerHandle &timer_handle)
```

Reply:
Also, I think the class pointer we are passing in is always `Buffer`, so that first parameter can become `Buffer *class_ptr`.